go.mod

```diff
@@ -63,6 +63,7 @@ require (
 	github.com/prometheus/client_model v0.6.1
 	github.com/prometheus/common v0.55.0
 	github.com/robfig/cron/v3 v3.0.1
+	github.com/sirupsen/logrus v1.9.3
```

Comment:
This might cause issues in the rebase -- can we do this without touching vendor?

Reply:
yeah, I can just switch to prints for the logging.